src/views/CreateShareStructure.vue

```diff
@@ -154,13 +151,17 @@ export default class CreateShareStructure extends Vue {
 
   private showShareStructureForm: boolean = false
   private addEditInProgress: boolean = false
-  private currentShareStructure: ShareClassIF | null = null
+  private currentShareStructure: ShareClassIF = null
 
   private activeIndex: number = -1
   private parentIndex: number = -1
   private shareId: string = ''
   private helpToggle: boolean = false
 
+  private get shareClasses (): ShareClassIF[] {
+    return this.getCreateShareStructureStep.shareClasses
+  }
```

Comment:
Another little helper so I didn't have to change all the code that used `shareClasses`.